core/src/com/unciv/models/ruleset/Building.kt

```diff
@@ -758,4 +758,8 @@ class Building : RulesetStatsObject(), INonPerpetualConstruction {
         }
         return false
     }
+
+    fun canBeHurried() : Boolean {
```

Comment:
This doesn't need to be a function in the building, we don't need a boolean for every unique a building can have